basic_module/account/src/traits/impls.rs

```diff
@@ -0,0 +1,220 @@
+use super::core::handles::*;
+use crate::account::get_storage_id;
+use crate::account::{
+    add_balance, get_balance, get_previous_balance, get_previous_sequence, get_sequence, sub_balance,
+};
+use crate::{get_context, Account, Action};
+use key::{public_to_address, verify_ed25519, Generator, KeyPair, Random};
+
+#[allow(dead_code)]
+pub struct Handler {}
+
+impl CheckTxHandler for Handler {
+    fn verify_transaction(tx: &Transaction) -> bool {
+        match &tx.action {
+            Action::Pay {
+                sender,
+                receiver: _,
+                quantity,
+            } => {
+                let sender_balance = get_balance(&sender);
+                let sequence = get_sequence(&sender);
+                if sender_balance.is_none()
+                    && sequence.is_none()
+                    && sender_balance.unwrap() < tx.fee + quantity
+                    && sequence.unwrap() <= tx.seq
+                {
+                    return false
```

Comment:
If what you want to test is the boolean expression and the determined value is also boolean, It seems clear to return the test formula directly